.github/workflows/.goreleaser.yaml

```diff
@@ -0,0 +1,38 @@
+project_name: helm
+
+builds:
+  - id: helm
+    main: ./cmd/helm
+    binary: helm
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
```

Comment:
for `GOARCH` we previously had:
```
darwin/amd64 darwin/arm64 linux/amd64 linux/386 linux/arm linux/arm64 linux/ppc64le linux/s390x linux/riscv64 windows/amd64 windows/arm64
```

ie. `linux/386`, `linux/arm`, `linux/ppc64le`, `linux/s390x linux/riscv64` need to continue to be built, but are missing here.